src/api/commons/middlewares/crud-nested.middlewares.js

```diff
@@ -1,7 +1,7 @@
 import { NotFoundError, ServerError } from '../http-errors';
 
 const read = (repository, useQuery) => async (req, res, next) => {
-  const { id, resourceId } = req.params;
+  const { id, resourceId } = req?.params || {};
```

Comment:
is there any way req.params isn't set when we call controllers.<function> ?
If yes, shouldn't we let it fail ?

Reply:
Fixed